Rakefile

```diff
@@ -6,7 +6,7 @@ require 'spree/testing_support/common_rake'
 
 RSpec::Core::RakeTask.new
 
-task :default => [:spec]
+task default: [:spec]
```

Comment:
Looking at the views, there are many places where the hash rocket syntax is used. So should we change it here?